barretenberg/cpp/src/barretenberg/api/api_ultra_honk.cpp

```diff
@@ -110,6 +110,8 @@ bool _verify(const bool honk_recursion_2, const std::filesystem::path& proof_pat
         verified = verifier.verify_proof(proof);
     }
 
+    verified ? info("Proof verified successfully") : info("Proof verification failed");
```

Comment:
I have never seen anyone do this in C++ - I'd say this is unidiomatic

Reply:
updated